src/core/Microsoft.PowerToys.Settings.UI/ViewModels/PowerPreviewViewModel.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.PowerToys.Settings.UI.Helpers;
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.PowerToys.Settings.UI.Views;
+
+namespace Microsoft.PowerToys.Settings.UI.ViewModels
+{
+    public class PowerPreviewViewModel : Observable
+    {
+        private const string ModuleName = "File Explorer Preview";
+
+        private PowerPreviewSettings Settings { get; set; }
+
+        public PowerPreviewViewModel()
+        {
+            Settings = SettingsUtils.GetSettings<PowerPreviewSettings>(ModuleName);
+
+            this._svgRenderIsEnabled = Settings.properties.IDS_PREVPANE_SVG_BOOL_TOGGLE_CONTROLL.value;
```

Comment:
[nit]Using `Settings.properties...` directly to set and read the value instead of keeping a local copy might be easier and avoid some extra asignments.

Reply:
what do you mean?